examples/analog/main.cpp

```diff
@@ -14,6 +14,11 @@ using namespace libm2k::devices;
 int main(int argc, char* argv[])
 {
 	M2k *ctx = m2kOpen();
+	if (!ctx) {
+		std::cout << "Connection Error: No ADALM2000 device available/connected to your PC." << std::endl;
+		return 0;
```

Comment:
`return 0` means success;
non-zero means failure;

it does not make much difference what return code you return; people usually check against zero; so `return 1` may be sufficient

Reply:
also, there doesn't seem to be any `return 0` at the end of this function;
curios: does C++ implicitly `return 0`?

Reply:
As far as I know if the control reaches the end of the main function, then it will implicitly return 0 .